Gruntfile.js

```diff
@@ -19,45 +19,28 @@
 
 'use strict';
 
-const webpack = require('webpack');
-const ProgressPlugin = require('webpack/lib/ProgressPlugin');
-
 module.exports = grunt => {
   require('load-grunt-tasks')(grunt);
 
-  const config = {
-    aws: {
-      port: 5000,
-    },
-    server: {
-      port: 8888,
-    },
-  };
-
   /* eslint-disable sort-keys */
   const dir = {
     app_: 'app',
     app: {
       demo: 'app/demo',
       ext: 'app/ext',
       page: 'app/page',
-      script: 'app/script',
       style: 'app/style',
-      templateDist: 'app/page/template/_dist',
     },
-    aws_: 'aws',
-    aws: {
-      s3: 'aws/s3',
-      static: 'aws/static',
-      templates: 'aws/templates',
+    dist: {
```

Comment:
We don't have anymore the 2 step compilation
- from src to deploy
- and from deploy to aws.

We now just have a single compilation target called `dist` (the `aws` folder renamed to be more generic). All the sources are compiled there